_posts/en/pages/faq/2016-01-21-rbf-faq.md

```diff
@@ -0,0 +1,305 @@
+providing a standard implementation that can fulfill user demand for
+this feature, it's possible there will be less incentive for miners to
+begin replacing any transaction with higher fee rate variants.
+
+## I heard Opt-in RBF was added with little or no discussion
+
+Recent RBF discussions going back to May 2015.
+
+Github:
+
+- Add first-seen-safe replace-by-fee logic to the mempool [#6176](https://github.com/bitcoin/bitcoin/pull/6176)
+- Scheduled full-RBF deployment [#6352](https://github.com/bitcoin/bitcoin/pull/6352)
+- nSequence-based Full-RBF opt-in [#6871](https://github.com/bitcoin/bitcoin/pull/6871)
+
+IRC meetings:
+
+- [2015-11-05][]
+- [2015-11-12][]
+- [2015-11-19][]
+- [2015-11-26][]
+
+There are many other logs for [#bitcoin-dev](http://bitcoinstats.com/) and [#bitcoin-core-dev](https://botbot.me/freenode/bitcoin-core-dev/).
+
+Mailing list discussions in no particular order:
+
+- [[bitcoin-dev] Opt-in Full Replace-By-Fee (Full-RBF)](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-November/011783.html)
+- [[Bitcoin-development] First-Seen-Safe Replace-by-Fee](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-May/008248.html)
+- [[Bitcoin-development] Cost savings by using replace-by-fee, 30-90%](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-May/008232.html)
+- [[bitcoin-dev] Significant losses by double-spending unconfirmed transactions](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-July/009420.html)
+- [[bitcoin-dev] BIP: Full Replace-by-Fee deployment schedule](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-June/009253.html)
+- [[Bitcoin-development] replace-by-fee v0.10.0rc4](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-February/007404.html)
+- [[bitcoin-dev] How wallets can handle real transaction fees](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-November/011685.html)
+
+Satoshi originally introduced [unconfirmed transaction replacement](https://github.com/trottier/original-bitcoin/blob/master/src/main.cpp#L434)
+but this was disabled due to a DOS attack (which Peter Todd fixed by requiring a higher fee for each replacement).
+
+## Can this be used to double spend against unprepared (old) wallets? Will all wallets have to update?
+
+Unconfirmed transactions can always be double spent, with or without RBF. Fraud
+is currently easy, and RBF doesn't change that.
+
+Wallets only have to update if they want to make use of opt-in RBF. This
+gives wallets a new dimension along which they can innovate and provide
+beneficial features to users.
+
+## Why not First-seen-safe Replace-by-fee (FSS-RBF)
+
+FSS-RBF means that transactions can only be modified if all previous
+outputs are fully spent. This prevents double spends, but it has three
+problems when it actually gets used:
+
+1. It results in increased transaction size because you must add a new
+   input to each replacement.
```

Comment:
To pick up https://github.com/bitcoin-core/website/pull/60#discussion_r50530922, maybe add something like:

> It results in increased transaction size because you must add a new input to each replacement*, which ultimately results in less network capacity.*